content/foundations/color.md

```diff
@@ -0,0 +1,193 @@
+
+![backgrounds-tertiary](https://user-images.githubusercontent.com/6951037/102642836-91931980-415e-11eb-933b-a86e5051c40a.png)
+
+
+#### Overlay + 4
+`bg-overlay`
+Use overlay background for elements positioned above all levels (absolute closest). For example, dropdowns, dialogs, select menus or popovers.
+
+![backgrounds-overlay](https://user-images.githubusercontent.com/6951037/102642832-8fc95600-415e-11eb-9b43-d3b6524a8793.png)
+
+---
+
+### Borders
+
+![borders-overview](https://user-images.githubusercontent.com/6951037/102643035-eb93df00-415e-11eb-8151-84e1c4d01be9.png)
+
+#### Primary
+`border-primary`
+
+Primary borders are for grouping elements. For example the border around cards or box components.
+
+![borders-primary](https://user-images.githubusercontent.com/6951037/102643029-ea62b200-415e-11eb-9cfc-d816ae65a4eb.png)
+
+#### Secondary
+`border-secondary`
+
+Secondary borders are supporting lines that separate or divide content like sections or list items.
+
+![borders-secondary](https://user-images.githubusercontent.com/6951037/102643034-eafb4880-415e-11eb-933a-8abb9cd3d018.png)
+
+```
+💡 Accessibility recommendations. 
+Some high contrast modes in browsers or systems remove backgrounds and rely only on borders to create boundaries. Use the full bundle to make sure these components look correct. Here’s an example.
+```
+
+// Note 🚨 Pending finding a screenshot from GH on a firefox browser that has high-contrast on 🚨
+
+### Text and icons
+
+**Primary**
+`text-primary`
+`icon-primary`
+
+Primary text and icons are the most visible foregrounds. The most common use of `text-primary`is default text like paragraphs, headings and labeling text. `icon-primary` is typically used as supporting symbol for text. Don't use text variables for icons, they have different accessibility requirements.
+
+**Secondary**
+`text-primary`
+`icon-primary`
+
+Use secondary text for words or paragraphs that provide secondary information or not a critical part of a primary workflow. 
+
+**Tertiary**
+`text-tertiary`
+`icon-tertiary`
+
+// Note 🚨 *Legacy and will be deprecated soon.* For muted text or disabled use `text-disabled` or `text-placeholder` instead. 
+
+**Disabled and placeholder**
+`text-disabled`
+`text-placeholder`
+
+
+**Links**
+`text-link`
+
+Links are clickable highlighted text.
+
+### States
+Functional group for interaction states. Each state has its own background, border, icon and text variable.States also have two levels of hierarchy. Primary and secondary. Primary is for most important interactions and cues. For example the selected state for pagination uses `-selected-primary` because it provides context about a whole view. Secondary states are for less important interactions. 
```

Comment:
```suggestion
Functional group for interaction states. Each state has its own background, border, icon and text variable. 

States also have two levels of hierarchy: primary and secondary. Primary is for most important interactions and cues. For example, the selected state for pagination uses `-selected-primary` because it provides context about a whole view. Secondary states are for less important interactions. 
```